assets/js/googlesitekit/api/cache.js

```diff
@@ -135,11 +135,11 @@ export const get = async ( key, cacheTimeToLive = null ) => {
 			// false-y if `0`, `null`, etc.)
 			if ( parsedData.timestamp && (
 				cacheTimeToLive === null || // Ensure the cached data isn't too old.
-        ( Date.now() / 1000 ) - parsedData.timestamp < cacheTimeToLive
+				Date.now() - parsedData.timestamp < cacheTimeToLive
```

Comment:
I see you've changed the timestamp to be stored in milliseconds, but comments and docs still refer to the `cacheTimeToLive` in seconds. Is this intentional?

Maybe we should add a `nowInSeconds()` helper to avoid doing needing to do the math everywhere the timestamp or TTL is checked?

Reply:
Nope! Sorry, I did some debugging here and left this. Let me change… 😄